stacks/jupyterhub-keycloak/keycloak-realm-config.yaml

```diff
@@ -0,0 +1,202 @@
+---
```

Comment:
If this is just copied does it make sense to move this into templates and share with e2e demo?

Reply:
Just the users and groups are the same - this demo doesn't need the UIF stuff, trino/superset clients etc. in the realm config so I have removed them.